tools/es5to6-files-per-user.js

```diff
@@ -0,0 +1,95 @@
+const path = require('path');
+const glob = require('glob');
+const { writeFileSync } = require('fs');
+
+const es5Dir = path.resolve(
+    __dirname,
+    '..',
+    'static',
+    'src',
+    'javascripts-legacy'
+);
+
+const files = glob.sync('**/*.js', {
+    cwd: es5Dir,
+});
+
+const filesPerHuman = (filesToCheck, humans) =>
+    Math.max(filesToCheck.length / humans.length);
+
+const commercial = [
+    'janua',
+    'JonNorman',
+    'regiskuckaertz',
+    'rich-nguyen',
+    'katebee',
+    'Calum-Campbell',
+    'ScottPainterGNM',
+];
+const commercialModules = files.filter(_ => _.includes('commercial'));
+const commercialFilesPerHuman = filesPerHuman(commercialModules, commercial);
+
+const platform = [
+    'sndrs',
+    'gtrufitt',
+    'SiAdcock',
+    'NataliaLKB',
+    'GHaberis',
+    'stephanfowler',
+];
+const platformModules = files.filter(_ => _.includes('projects/common/utils'));
+const platformFilesPerHuman = filesPerHuman(platformModules, platform);
+
+const misc = [
+    'desbo',
+    'akash1810',
+    'dominickendrick',
+    'davidfurey',
+    'jranks123',
+    'ShaunYearStrong',
+    'jfsoul',
```

Comment:
Why are Jon and Gustav part of the misc, and not part of platform? They are very much part of the platform team 😄